docs/man/docker-images.1.md

```diff
@@ -34,7 +34,9 @@ versions.
    Show all images (by default filter out the intermediate image layers). The default is *false*.
 
 **-f**, **--filter**=[]
-   Provide filter values (i.e., 'dangling=true')
+   Provide filter values. Valid filters:
```

Comment:
@moxiegirl  dangling=true was an existing feature and is actually unrelated to this labels PR.  I realize that is quite confusing in this message.  dangling=true refers to finding untagged images with no parents.  label=... means find an image based off of labels.  What would be a good way to phrase this?